### DISCUSSION TITLE
Automatic Decoding and Submitting Captcha?

### DISCUSSION THREAD

**Discussion by shivbuyya** [pallupz/covid-vaccine-booking#242] [Ideas]
Hi Pallupz,

I am new to Python, why don't you try automatic decoding of Captcha and submission. This will be very Helpful, because sometime we may not be infront of computer to enter it manually. I am sure you know it is required, It will be great if this feature is implemented.

**Comment by pallupz** [comment]
Hi - There are other repos that do that. I do not want to include it as it might be legally grey area and I do not want to find out.  This repo got way more attention that I ever thought it would to take that "risk".

**Reply by shivbuyya** [reply]
Ok, Anyways this is good enough for purpose to be solved. I see only missing thing is automation of captcha reading. That's fine, I respect your work done here and decision on proceeding further.

**Reply by AlakmarShafin** [reply]
I have decoded Captcha (without image recognition ) it just needs some extra libs:
time
requests
bs4,BeautifulSoup
JSON
base64
and I am using it with this project, it's working fine.
full project with auto captcha
https://github.com/AlakmarShafin/Cowin-covid-Vaccination-slot-booking-automation
